yarn-project/end-to-end/src/canary/browser.ts

```diff
@@ -112,6 +112,24 @@ export const browserTestSuite = (setup: () => Server, pageLogger: AztecJs.DebugL
       await deployTokenContract();
     }, 60_000);
 
+    it("Can access a wallet's addresses", async () => {
+      const result = await page.evaluate(
+        async rpcUrl => {
+          const { createPXEClient: createPXEClient } = window.AztecJs;
+          const pxe = createPXEClient(rpcUrl!);
+          const [wallet] = await AztecJs.getSandboxAccountsWallets(pxe);
+          const completeAddress: AztecJs.CompleteAddress = wallet.getCompleteAddress();
+          const address = wallet.getAddress();
+          const addressString = completeAddress.toString();
+          return [completeAddress, address, addressString];
+        },
+        PXE_URL,
+        (await getTokenAddress()).toString(),
+        TokenContractAbi,
+      );
+      expect(result).toEqual(initialBalance);
+    });
+
```

Comment:
test fails without the updated import!

https://app.circleci.com/pipelines/github/AztecProtocol/aztec-packages/13274/workflows/b0e79aa0-1e61-4a96-9a0f-ea8c9aa2ed58/jobs/543678